Client/Frontend/Widgets/SimpleHighlightCell.swift

```diff
@@ -180,9 +180,6 @@ class SimpleHighlightCell: UITableViewCell {
         if let bookmarked = site.bookmarked, bookmarked {
             self.descriptionLabel.text = "Bookmarked"
             self.statusIcon.image = UIImage(named: "context_bookmark")
-        } else {
-            self.descriptionLabel.text = "Visited"
```

Comment:
You still gotta keep the description label text `Visited` But just remove the timestamp itself.